doc/source/conf.py

```diff
@@ -1,6 +1,10 @@
 """Sphinx documentation configuration file."""
 from datetime import datetime
 
+from sphinx.builders.latex import LaTeXBuilder
+
+LaTeXBuilder.supported_image_types = ["image/png", "image/pdf", "image/svg+xml"]
```

Comment:
Does this line allow us to include SVG images in LaTeX? Never used this `LaTeXBuilder` class before...

Reply:
Yes. This allow us to download all the images from the link and keep it as SVG images inside the folder where all the latex files are. May be later we can add the SVG images directly from these folder.